ADAPTERS.md

```diff
@@ -1,28 +1,98 @@
+
+We are open to contributions to make this process easier.
+
+An example adapter lives [here](./apps/server/src/test-adapter/index.ts). Each adapter needs to export an adapterMapObject. These adapterMapObjects are imported and setup by someone hosting the UCW in [adapterSetup.ts](./apps/server/src/adapterSetup.ts). If the test adapterMapObject doesn't support a property you need to use for your adapter, then you'll need to add it to the test example adapter to ensure it continues to get support.
+
+This repo accesses adapter-specific logic in [adapterIndex.ts](./apps/server/src/adapterIndex.ts)
+
+## Test institutions
+
+Adding a test institution to the [default institution list](./apps/server/cachedDefaults/ucwInstitutionsMapping.json) allows you to test your adapter. 
+
+For now, add this manually to the [ucwInstitutionsMapping.json](./apps/server/cachedDefaults/ucwInstitutionsMapping.json) file. In the future, UCP will have a way to manage and sync your aggregator and aggregator/institution mappings via an online service.
+
+```typescript
+const testInstitution = {
+  "name": "My Test Bank",
+  "id": "00000000-0000-0000-0000-000000000000",
+  "keywords": ["test bank", "text bank", "test bakn"],
+  "logo": "https://universalconnectproject.org/images/ucp-logo-icon.svg",
+  "url": "https://my-test-bank.com",
+  "is_test_bank": true,
+  "routing_numbers": [],
+  "myAggregatorName": {
+    "id": "myAggregatorId",
+    "supports_aggregation": true,
+    "supports_history": true,
+    "supports_identification": true,
+    "supports_oauth": true,
+    "supports_verification": true
+  }
+};
+```
+## Monorepo 
+
+Because this repo is a monorepo, here are some caveats to consider with regard to adapter creation.
+
+### Aggregator adapter package and Turborepo
+
+See [the ADAPTERS readme](ADAPTERS.md) for more details on what aggregator adapter packages are.
+
+There are two ways to create a shared package within a Turborepo monorepo. You can create Just-in-Time (JIT) packages or compiled packages. Please see [Turborepo's Documentation](https://turbo.build/repo/docs/core-concepts/internal-packages#compiled-packages) for more detailed information.
+
+Adapter Packages are considered compiled packages, and even though end-users will install your adapter package using NPM, the preferred way what we've chosen to create and test an adapter package is to use it as a compiled package.
+
+Below you'll find some tips on how to successfully run the UCW with a compiled package.
+
+We will use [the MX Adapter](https://github.com/Universal-Connect-Project/ucw-adapter-mx/tree/main/packages/mx-adapter) as a reference.
+
+#### Development tips
+
+When you create an adapter package, you'll need to install it in your monorepo workspace using the following command:
+
+```bash
```

Comment:
Someone creating an adapter package doesn't need to do this right? Isn't this only for those hosting?

Reply:
That is intended for anyone creating a new adapter. In order to test it with `app/server`, they need to add it to the server package file. I'll add some more details to make that more clear.